BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -46,14 +46,16 @@
 #define UPLOAD_API GLOBAL_API_PREFIX @"/uploads"
 #define UPLOAD_STATUS_API GLOBAL_API_PREFIX @"/uploadstatuses"
 
-static NSString * const kSBBUploadAPI =                 UPLOAD_API;
-static NSString * const kSBBUploadCompleteAPIFormat =   UPLOAD_API @"/%@/complete";
-static NSString * const kSBBUploadStatusAPIFormat =     UPLOAD_STATUS_API @"/%@";
+NSString * const kSBBUploadAPI =                 UPLOAD_API;
+NSString * const kSBBUploadCompleteAPIFormat =   UPLOAD_API @"/%@/complete";
+NSString * const kSBBUploadStatusAPIFormat =     UPLOAD_STATUS_API @"/%@";
 
 static NSString *kUploadFilesKey = @"SBBUploadFilesKey";
 static NSString *kUploadRequestsKey = @"SBBUploadRequestsKey";
 static NSString *kUploadSessionsKey = @"SBBUploadSessionsKey";
```

Comment:
Only needed the API strings to be extern so they'd be available for the unit tests.